### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ducphamle2**

### CODE REVIEW

**Review by im6h — APPROVED** [review]

**Review by phutxorai — DISMISSED** [review]
lgtm

**Review by im6h — APPROVED** [review]
Lgtm

**Review by haunv3 — DISMISSED** [review]
lgtm

**Review by haunv3 — APPROVED** [review]

**Review by im6h — APPROVED** [review]
lgtm

**Review by tubackkhoa — APPROVED** [review]

**Review by phutxorai — APPROVED** [review]
lgtm

File: src/config/pools.ts [hunk 1]
```diff
@@ -89,7 +90,9 @@ export const getPair = (
 
   return pairs.find(
     (pair) =>
-      pair.asset_denoms[0] === asset_denoms[0] &&
-      pair.asset_denoms[1] === asset_denoms[1]
+      (pair.asset_denoms[0] === asset_denoms[0] &&
```
[the remark this answers is not in the document]
Comment (phutxorai) [reply src/config/pools.ts:93]: @haunv3 should we update like that ?

**Pull request merged**